packages/static-module-record/src/transform-analyze.js

```diff
@@ -32,7 +32,16 @@ const makeCreateStaticRecord = transformSource =>
       // Comment out the shebang lines.
       moduleSource = `//${moduleSource}`;
     }
-    const scriptSource = transformSource(moduleSource, sourceOptions);
+    let scriptSource;
+    try {
+      scriptSource = transformSource(moduleSource, sourceOptions);
+    } catch (err) {
+      const moduleLocation = url ? `"${url}"` : '<unknown>';
```

Comment:
Quoting strings, generally.
```suggestion
      const moduleLocation = url ? JSON.stringify(url) : '<unknown>';
```